### DISCUSSION TITLE
manual homing

### DISCUSSION THREAD

**Discussion by yuuahmad-aii** [grblHAL/core#627] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/d70dcf69-2780-463a-8054-cb8dea14d1d9)
![Cuplikan layar 2024-11-29 153812](https://github.com/user-attachments/assets/99b4e55a-a863-4092-b11a-d527fa61658f)

based on the 2 lines that I marked, if I set $22=33 the $H command on mdi should be able to homing the axis on the machine without having to move the axis.

but it turns out that my machine still moves homing. even though my machine does not have a limit switch

what I want to aim for is
1. do manual homing. because my machine does not have a limit switch
2. when manual homing is done, the world coordinate position must be 0

can anyone help me?

**Comment by terjeio** [comment, marked as answer]
Set $44, $45 and $46 to 0 to enable manual homing for all axes.

**Reply by yuuahmad-aii** [reply]
thank you.
now, it's working as i want
